tests/x11/dolphin.pm

```diff
@@ -1,23 +1,74 @@
 # SUSE's openQA tests
 #
 # Copyright © 2009-2013 Bernhard M. Wiedemann
-# Copyright © 2012-2017 SUSE LLC
+# Copyright © 2012-2018 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Startup of dolphin
-# Maintainer: Oliver Kurz <okurz@suse.de>
+# Summary: Start dolphin and do some file operations
+# Maintainer: Fabian Vogt <fvogt@suse.de>
 
 use base "x11test";
 use strict;
 use testapi;
 
 sub run {
-    x11_start_program('dolphin');
-    send_key "alt-f4";
+    x11_start_program 'dolphin';
+
+    # Go to ~/Documents
+    assert_and_click 'dolphin_icon_documents';
+    assert_screen 'dolphin_documents_empty';
+
+    # Create a new folder
+    send_key 'f10';
+    type_string 'stuff';
+    assert_screen 'dolphin_new_folder';
+    send_key 'ret';
+    # Enter the new folder
+    send_key 'ret';
+
+    # Context menu: "Create new" -> "Text file"
+    assert_and_click 'dolphin_stuff_empty', 'right';
+    my $create_new = assert_screen 'dolphin_create_new';
+
+    my $lastarea = $create_new->{area}->[-1];
```

Comment:
You can use `xy` function from lib/caasp_controller.pm for getting coordinates

Reply:
Why is that in caasp_controller? If it were somewhere more fitting I would use it...

Reply:
you may suggest PR which move this function to utils.pm :)

Reply:
A new parameter "hide_mouse" to assert_and_click probably makes more sense